src/styles.scss

```diff
@@ -21,11 +21,8 @@ button{
 
 body.dark-theme {
   background: $dark-theme-background;
+  color: $dark-theme-text-color;
 
```

Comment:
If you set it above you will have to set the svg fill color here to, don't you?

Reply:
It should work as it is, but for consistency reasons I can add it here too.